src/main/resources/com/google/jenkins/plugins/computeengine/InstanceConfiguration/config.jelly

```diff
@@ -41,12 +41,27 @@
             <f:entry title="${%Number of Executors}" field="numExecutorsStr">
                 <f:textbox default="1"/>
             </f:entry>
+            <f:entry title="${%Remote Location}" field="remoteFs">
+                <f:textbox default=""/>
+            </f:entry>
             <f:entry field="preemptible" title="${%Preemptible?}">
                 <f:checkbox/>
             </f:entry>
-             <f:entry field="minCpuPlatform" title="${%Minimum Cpu Platform}">
-                 <f:select/>
-             </f:entry>
+            <f:entry field="windows" title="${%Windows?}">
+                <f:checkbox/>
+            </f:entry>
+            <f:entry field="windowsUsername" title="${%Windows Username}">
```

Comment:
You can embed the windows specific fields inside an if tag so they aren't displayed unless necessary. See: http://commons.apache.org/proper/commons-jelly/tags.html#core:if

Reply:
I can't seem to access the windows variable. I am tracking this issue/feature request; agreed it would enhance usability.